docs/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
```

Comment:
Maybe we don't need this Makefile?  Would it be possible to just call sphinx directly (using `python -m sphinx`) from the `uvloop/Makefile`?